fix(ddx): update wording (#4775)
scaleway/docs-content: pages/dedibox-kvm-over-ip/how-to/use-bmc.mdx

---
meta:
  title: Understanding the Baseboard Management Controller (BMC) on dedicated servers

>   description: Learn how to use the BMC for remote management, monitoring, and troubleshooting of dedicated servers.

Author: ```suggestion
  description: Learn how te the BMC can help you with remote management, monitoring, and troubleshooting of dedicated servers.
```

content:
  h1: Understanding the Baseboard Management Controller (BMC) on dedicated servers

>   paragraph: This page explains how to use the Baseboard Management Controller (BMC) for remote management, monitoring, and troubleshooting of dedicated servers.

Author: ```suggestion
  paragraph: Learn how te the BMC can help you with remote management, monitoring, and troubleshooting of dedicated servers.
```
